### DISCUSSION TITLE
Cannot get emails, CSS, or links to work

### DISCUSSION THREAD

**Discussion by philiphyatt** [community/community#65146] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I am trying to host a website through GitHub Pages, but I am running into some issues. No matter what I do, I cannot get my HTML page to reference CSS or image files. I have watched many tutorials online and I am almost certain that I am doing everything correctly. I cannot for the life of me figure out what the issue is, but the CSS and JPGs just will not show up. Also, none of the links to other HTML pages work, even though they have all successfully been uploaded. Any suggestions for what I might be doing wrong? 

My repo is: https://github.com/philiphyatt/GameOfTheYear.git

**Comment by nirav-gajera** [comment]
You are trying to reference the CSS and image files from the root directory of your repo, but GitHub Pages expects you to reference them from the `/public` directory.

To fix this, you need to move the CSS and image files to the` /public` directory. You can do this by running the following commands:

```
cd GameOfTheYear
mv style.css public/style.css
mv gameofyear.jpg public/gameofyear.jpg
```
Once you have moved the files, you need to update the links in your HTML page to point to the new location of the files. For example, if you were previously linking to style.css, you would need to change the link to public/style.css.

**Comment by philiphyatt** [comment]
Hello, thanks for the response. Sorry if this is a dumb question (I'm very
new to this), but how do I run commands on github? As in, where do I input
the code you just provided?
